app/models/invoice.rb

```diff
@@ -338,6 +341,10 @@ def generate_organization_sequential_id
     result
   end
 
+  def assign_payment_dispute_lost_at
```

Comment:
Can we rather set `payment_dispute_lost_at` using the common approach in the Lago app:
```
def mark_as_disputed!(timestamp = Time.current)
    self.payment_dispute_lost_at ||= timestamp
    self.payment_disputed = true
  end
```
in terms of readability, I would rather avoid callbacks if possible. This method can be called wherever we need it.